nativescript-imagepicker/albums.ios.js

```diff
@@ -40,3 +49,34 @@ function done(args) {
     page.bindingContext.done();
 }
 exports.done = done;
+function albumsPageFactory() {
```

Comment:
Yep, I leave the xml intact hoping it would be an alternative soon and the method just reverted.